src/rgtk.rs

```diff
@@ -60,8 +60,9 @@ extern crate "cairo-sys" as cairo_ffi;
 
 extern crate glib;
 
+pub use glib::Type;
+pub use glib::{GValue, GValuePublic};
```

Comment:
Oh, I meant not reexporting glib seems cleaner, doesn't it? Any GTK program would depend on `glib` implicitly so it might as well do `extern crate glib` and import types from there. Need to add a `glib` dependency to the examples `Cargo.toml` too.